onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -6915,7 +6915,8 @@ def aten_roll(self: TTensor, shifts: INT64, dims: Sequence[int] = ()) -> TTensor
     elif self.shape[0] == 0:  # empty tensor
         return self
     else:
-        if isinstance(dims, tuple) and len(dims) == 0:  # Empty list
+        # NOTE: In pytorch, default value of dims is an empty list.
+        if isinstance(dims, Sequence) and len(dims) == 0:  # Empty sequence
```

Comment:
Now that I look at this, we shouldn’t even need this isinstance check because dims is always a sequence